input/pagecontent/general-requirements.md

```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
 For example, the profile [AU Core Medication](StructureDefinition-au-core-medication.html) constrains the terminology choices for `Medication.code.coding` defined in [AU Base Medication](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-medication.html) to support [Australian Medication](https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1) and [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-pbs-item.html) as indicated by flagging Must Support on those two terminology slices.
 
 When claiming conformance to the AU Core Medication profile: 
-- AU Core Responders **SHALL** be capable of populating `Medication.code.coding` with a code from [Australian Medication](https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1) or [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-pbs-item.html), or both, if a coded value is available.
-- AU Core Requesters **SHALL** be capable of meaningfully processing a Medication resource with `Medication.code.coding` with both.
-- Systems **MAY** support populating and processing other terminologies but this is not a requirement of AU Core.
+- AU Core Responders **SHALL** correctly populate `Medication.code.coding` with either a code from [Australian Medication](https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1) or [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-pbs-item.html), or both, if a coded value is known.
+- AU Core Requesters **SHALL** accept a Medication resource without error if `Medication.code.coding` present with either a code from [Australian Medication](https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1) or [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-pbs-item.html), or both.
+
+Systems **MAY** populate and accept other code systems but this is not a requirement of AU Core.
+
+### Missing Data
+
+There are situations when information for a  data element is missing and the  system does not know reason for the absence of data. 
+
+When an element definition is *optional* (minimum cardinality = 0), including elements labelled *Must Support*, an
+- AU Core Responder **SHALL NOT** populate the element in the resource when a value is not known.  
+
+When element definition is *Mandatory* (minimum cardinality > 0), 
+- AU Core Responder **SHALL** correctly populate the element *even when* the system does not know a value or the reason a the value is absence. 
```

Comment:
Fix "a the". Also need to make some minor grammar adjustments; suggest something like: 
- AU Core Responder **SHALL** correctly populate the element *even when* the system does not know a value or the reason for its absence.

Reply:
This text is no longer present.  but re-wrote slightly what is in master to improve it in this branch.